Gemfile

```diff
@@ -52,7 +52,7 @@ gem 'null_logger'
 gem 'plek', '1.9.0'
 gem 'diffy', '3.0.6'
 
-gem 'redis', '3.0.7', require: false # Only used in some importers
+gem 'redis', '3.2.1', require: false # Only used in some importers
```

Comment:
I don't think we need to explicitly require / pin this. It's required by Sidekiq and `mlanett-redis-lock`, both of which are explicitly required here.